src/win32/sys_win32.h

```diff
@@ -50,4 +50,4 @@ char** PE32_GetStrTable(void *buff, int len, sharedlib_data_t *text);
 
 extern WinVars_t g_wv;
 extern byte cod4_plt[8192];
-#define IDI_ICON1                       1
+#define IDI_ICON1                       1
```

Comment:
Only change is the removal of a newline, why?

Reply:
Likely my editor?